subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/fingerprint/ConfigurationCacheFingerprintWriter.kt

```diff
@@ -339,6 +344,14 @@ class ConfigurationCacheFingerprintWriter(
         }
     }
 
+    private
+    fun reportExternalProcessOutputRead(executable: String) {
+        reportInput(consumer = null, documentationSection = null) {
```

Comment:
:x: This should link to the documentation section added in this very PR

Reply:
Nice catch, forgot about that. Fixed now.